Gems/LyShine/Code/Source/Animation/UiAnimationSystem.cpp

```diff
@@ -726,24 +726,27 @@ void UiAnimationSystem::StillUpdate()
 //////////////////////////////////////////////////////////////////////////
 void UiAnimationSystem::ShowPlayedSequencesDebug()
 {
-    //f32 green[4] = {0, 1, 0, 1};
-    //f32 purple[4] = {1, 0, 1, 1};
-    //f32 white[4] = {1, 1, 1, 1};
+    constexpr f32 green[4] = {0, 1, 0, 1};
+    constexpr f32 purple[4] = {1, 0, 1, 1};
+    constexpr f32 white[4] = {1, 1, 1, 1};
     float y = 10.0f;
     AZStd::vector<AZStd::string> names;
 
+    //TODO: needs an implementation
+    auto Draw2dLabel = [](float /*x*/,float /*y*/,float /*depth*/,const f32* /*color*/,bool /*center*/, const char* /*fmt*/, ...) {};
+
     for (PlayingSequences::iterator it = m_playingSequences.begin(); it != m_playingSequences.end(); ++it)
     {
         PlayingUIAnimSequence& playingSequence = *it;
 
-        if (playingSequence.sequence == NULL)
+        if (playingSequence.sequence == nullptr)
         {
             continue;
         }
 
         AZ_Assert(false,"gEnv->pRenderer is always null so it can't be used here");
-        //const char* fullname = playingSequence.sequence->GetName();
-        //gEnv->pRenderer->Draw2dLabel(1.0f, y, 1.3f, green, false, "Sequence %s : %f (x %f)", fullname, playingSequence.currentTime, playingSequence.currentSpeed);
+const char* fullname = playingSequence.sequence->GetName();
+Draw2dLabel(1.0f, y, 1.3f, green, false, "Sequence %s : %f (x %f)", fullname, playingSequence.currentTime, playingSequence.currentSpeed);
```

Comment:
Why ucomment this code if Draw2dLabel doesn't have an implementation?

Also the whitespace is off here

Reply:
my guess is that it was done because of some unused variables, instead, just comment the unused variables